app/experimenter/templates/experiments/section_overview.html

```diff
@@ -5,14 +5,18 @@
 {% endblock %}
 
 {% block section_content %}
-  <p><strong>Experiment Owner:</strong> {{ experiment.owner }}</p>
+  <p><strong>Description</strong></p>
+  <p>{{ experiment.short_description|urlize|linebreaks }}</p>
+  <p><strong>Experiment Owner</strong></p>
+  <p>{{ experiment.owner }}</p>
   {% if experiment.engineering_owner %}
-    <p><strong>Engineering Owner:</strong> {{ experiment.engineering_owner }}</p>
+    <p><strong>Engineering Owner</strong></p>
+    <p>{{ experiment.engineering_owner }}</p>
   {% endif %}
-  {{ experiment.short_description|urlize|linebreaks }}
-  <p><strong>Public Name:</strong></p> 
-  {{ experiment.public_name| linebreaks }}
+  <p><strong>Public Name:</strong></p>
+  {{ experiment.public_name|linebreaks }}
   <p><strong>Public Description:</strong> </p>
-  {{ experiment.public_description| linebreaks }}
+  {{ experiment.public_description|linebreaks }}
```

Comment:
Didn't think of it when working on this earlier and not entirely sure about the use case for public description, but would we want "urlize" here too or would public descriptions ever contain a link?

Reply:
Yep good catch

Reply:
Public description is the text that will go in `about:studies`.